docs/changelogs/CHANGELOG-2.26.md

```diff
@@ -0,0 +1,320 @@
+- Fix VPA admission-controller PDB blocking evictions ([#13515](https://github.com/kubermatic/kubermatic/pull/13515))
+- Improve helm repository prefix handling for system applications; only prepend `oci://` prefix if it doesn't already exist in the specified URL ([#13336](https://github.com/kubermatic/kubermatic/pull/13336))
+- Installer does not validate iap client_secrets for grafana and alertmanager the same way it does for encryption_key ([#13315](https://github.com/kubermatic/kubermatic/pull/13315))
+- Restore missing bgpconfigurations CRD in Canal 3.27 ([#13505](https://github.com/kubermatic/kubermatic/pull/13505))
+- Update Canal 3.27 to 3.27.4 and Canal 3.28 to 3.28.1 ([#13625](https://github.com/kubermatic/kubermatic/pull/13625))
+- When the cluster-backup feature is enabled, KKP will now reconcile a ConfigMap in the `velero` namespace in user clusters. This ConfigMap is used to configure the restore helper image in order to apply KKP's image rewriting mechanism ([#13471](https://github.com/kubermatic/kubermatic/pull/13471))
+
+
+### Updates
+
+- Update `kubermatic/util` to Alpine 3.19 ([#13187](https://github.com/kubermatic/kubermatic/pull/13187))
+- Bump Seed MLA Grafana to 10.4 ([#13223](https://github.com/kubermatic/kubermatic/pull/13223))
+    - Update seed-MLA Grafana to 10.4.1
+    - Update seed-MLA Grafana dashboards: more consistent styling, do not use deprecated Chart panels anymore
+    - Remove all custom Grafana plugins (`grafana-piechart-panel`, `farski-blendstat-panel`, `michaeldmoore-multistat-panel` and `vonage-status-panel`): most are deprecated or soon defunct and none of the KKP dashboard use any of these panel types
+- Bump usercluster/metering Prometheus to 2.51.1 ([#13306](https://github.com/kubermatic/kubermatic/pull/13306))
+    - Update usercluster Prometheus to 2.51.1
+    - Update metering Prometheus to 2.51.1
+- Apply OCI labels to all KKP container images ([#13210](https://github.com/kubermatic/kubermatic/pull/13210))
+    - Improve labels on KKP container images.
+    - Update container images to Alpine 3.19
+- Update MLA Alertmanager Proxy helm chart ([#13222](https://github.com/kubermatic/kubermatic/pull/13222))
+    - Update Alertmanager Authorization Envoy to v1.29.2
+    - Improve alertmanager-proxy Helm Chart: do not require root permissions, drop capabilities and make logging/ports configurable
+- Add support for Canal 3.28, deprecate Canal 3.25 ([#13504](https://github.com/kubermatic/kubermatic/pull/13504))
+- Allows KKP administrator to interface thanos query with thanos-sidecar to get full benefit of using thanos ([#13482](https://github.com/kubermatic/kubermatic/pull/13482))
+- Cilium kubeProxyReplacement values strict, partial, probe, and disabled have been deprecated, please use true or false instead ([#13291](https://github.com/kubermatic/kubermatic/pull/13291))
```

Comment:
`strict`, `partial`, `probe` and `disabled`

(highlight as code, no comma after probe)